### PR TITLE
Title of current tab being sent to clients.

### PR DESCRIPTION
**Pull request by alir128**
#39

Title from host to client over data channels.

image

@Lakshya2610 : Client App code needs to be updated to add the changes.
@luu-alex : New UI might need the new code from popup.js and popup.html to add these changes.

### CODE REVIEW

**Comment by Lakshya2610** [comment]
I'll handle the client side for this update

**Comment by alir128** [comment]
> I'll handle the client side for this update

That would be great! Thanks

File: extension/src/js/background/index.js [hunk 1]
```diff
@@ -120,6 +153,17 @@ chrome.tabs.onRemoved.addListener(function(tabId, removed) {
     }
 });
 
+function addTitleListener() {
+    chrome.tabs.onUpdated.addListener(function(tabId, changeInfo) {
+        if(changeInfo.title && tabId === tabID) {
```
Comment (luu-alex) [inline extension/src/js/background/index.js]: Line #158 in background/index.js, tabId and tabID is confusing to me, is there a way we can have a different var name?

**Comment by luu-alex** [comment]
Thanks for the heads up

**Pull request merged**